### ISSUE TITLE
Missing packages when installing on Fedora

### ISSUE THREAD

**Issue by johanneskastl** [jvoss/ansible-role-netbox#58]
I used this role successfully on Fedora 37 (see #57 ).

I had to add some packages to get it working, see this playbook:

```yaml
- name: 'Install netbox'
  hosts: 'all'
  gather_facts: true
  become: true

  pre_tasks:

    - name: Install cronie
      ansible.builtin.package:
        name: cronie
        state: present

    - name: Install acl
      ansible.builtin.package:
        name: acl
        state: present

    - name: Install python3-django
      ansible.builtin.package:
        name: python3-django
        state: present

  roles:
    - role: 'jvoss.netbox'
```

If desired I can add a PR to make the role install those packages on Fedora.

**Comment by jvoss** [comment]
Thanks @johanneskastl! A PR would be greatly appreciated.

**Comment by johanneskastl** [comment]
I included a fix in #57, see commit 6299890

**Comment by jvoss** [comment]
Resolved by #57 . Thanks again!
